### PR TITLE
refactor: pseudo events modularisation

### PR DESCRIPTION
**Pull request by MapTo0**
- PsuedoEvents is now a module that exports several reusable methods for identifying keydown

### CODE REVIEW

File: packages/main/src/ListItem.js [hunk 1]
```diff
@@ -94,6 +94,8 @@ class ListItem extends ListItemBase {
 	onBeforeRendering() {}
 
 	onkeydown(event) {
+		super.onkeydown(event);
```
Comment (vladitasev) [inline packages/main/src/ListItem.js]: Why is this needed now and wasn't before?

Comment (MapTo0) [reply packages/main/src/ListItem.js]: the logic was handled in psuedo events `onsap...` which was propagated from the `ListItem` to `ListItemBase`. As we are not having these event anymore, we should always call parent's `keydown` if we want to have propagation in the inheritance chain
